.github/workflows/docs.yml

```diff
@@ -2,10 +2,20 @@ name: Documentation Linter
 
 on:
   pull_request:
+    paths:
```

Comment:
This change requires removing that workflow from required as it's not being run right now.